packages/pharos-site/static/styles/global.scss

```diff
@@ -1,5 +1,6 @@
 @use '@ithaka/pharos/lib/styles/pharos';
-@use '@ithaka/pharos/lib/utils/scss/mixins';
```

Comment:
this should not be commented out
but scss-loader isn't able to find this file and i'm not sure why 😕

Reply:
fixed
needed to include the "_" in the export